bm7/bm7.c

```diff
@@ -1,12 +1,11 @@
 #include "strucs.h"
 
-somestruct foo(void);
-void bar(somestruct);
+somestruct foo()
```

Comment:
This change is not correct, the driver.c and bm7.c files are not actually mixed up (even though driver.c is perhaps badly named in this case since it is not really a driver). In fact the bm7.c is the file that needs to be compiled with the tested compiler (even though it contains the main method - that is deliberate! It is testing that the compiler will correctly construct the structure "in place" when it issues a call to the 'bar' function. However, the 'bar' function itself is not part of the benchmark (notice that it simply returns immediately).